src/routes/dashboard/+page.svelte

```diff
@@ -1 +1,8 @@
-<h1>User dashboard</h1>
\ No newline at end of file
+<script>
+	import { authStore } from '$lib/store/store';
+</script>
+
+{#if !$authStore.loading}
+	<h1>User dashboard</h1>
+    <p>Welcome, user email: {$authStore.user}</p>
```

Comment:
I will try to config vs code to run prettier on each save. Until then (or if you're using another ide), please make sure to do auto styling with prettier.

Reply:
I don't know why code looks here like that. In my IDE, the indentation is correct. I ran the prettier again and pushed. Hoe it is fixed now

Reply:
Interesting. Yes, now it's better